Formula/nettle.rb

```diff
@@ -26,9 +26,12 @@ def install
                                           "get_pc_thunk|(_*chkstk_darwin)"
     end
 
+    args = "--build=aarch64-apple-darwin#{`uname -r`.to_i}" if Hardware::CPU.arm?
```

Comment:
This won't work as expected; it leaves `args` undefined if `Hardware::CPU.arm?` is false, and splatting a single string may have unexpected results.

Can you follow the example in #58092? Define an empty array first, then add something to that array `if Hardware::CPU.arm?`. Then you can use the array with a splat in the `"configure"` call.

Reply:
Good call, thanks :) Fixed in 11f3f2a